test/spec/expose.js

```diff
@@ -29,18 +29,12 @@ asyncTest("AMD support", 1, function() {
     baseUrl: "../",
 
     paths: {
-      underscore: "bower_components/underscore/underscore",
-      jquery: "bower_components/jquery/dist/jquery",
-      backbone: "bower_components/backbone/backbone"
+      underscore: "node_modules/underscore/underscore",
+      jquery: "node_modules/jquery/dist/jquery",
+      backbone: "node_modules/backbone/backbone"
     },
 
     shim: {
-      underscore: { exports: "_" },
-      backbone: {
-        deps: ["jquery", "underscore"],
-        exports: "Backbone"
-      },
-
       "backbone.layoutmanager": useLM || {
```

Comment:
Are you asking about line 38? If so, try removing it.  If the tests still pass, we don't need it ;-)